ledger/internal/eval_test.go

```diff
@@ -22,7 +22,6 @@ import (
 	"errors"
 	"fmt"
 	"math/rand"
-	"reflect"
```

Comment:
nice!

Reply:
Don't get too excited, I just moved the single test that used reflection into another package. But it still uses reflection.

I don't worry too much about using reflection in tests, do you?  Makes for easy deep equality checks.

Reply:
I'm pretty sure testify uses DeepEqual internally.